app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/controls/filtersControl/filtersItem/filtersItem.jsx

```diff
@@ -41,13 +43,20 @@ export class FiltersItem extends PureComponent {
           onChange={onChange}
           circleAtTop
         >
-          <FilterName search={search} filter={filter} userId={userId} showDesc={false} />
+          <FilterName
+            search={search}
+            filter={filter}
+            userId={userId}
+            showDesc={false}
+            editable={false}
```

Comment:
It's isn't mistake? `editable` here always should be `false`?

Reply:
Yes, you are right, i have corrected it. Thanks for the comment :)

Reply:
Not a mistake. We need to hide "edit" icon for this component unconditionally only in this case.